Survey/Survey/Sources/Data/Repositories/LoginRepository.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LoginRepository.swift
+//  Survey
+//
+//  Created by Taher on 4/11/22.
+//
+
+import Combine
+
+final class LoginRepository: LoginRepositoryProtocol {
+
+    let network: LoginNetworkAPI
+
+    init(network: LoginNetworkAPI) {
+        self.network = network
+    }
+
+    func getLoginToken(email: String, password: String) -> AnyPublisher<LoginTokenApi?, Error> {
```

Comment:
In Khanh's IC project, he has an excellent way of defining AnyPublisher and AnyCancellable. What do you think of using his idea?

```
typealias Observable<T> = AnyPublisher<T, Error>

typealias CancelBag = Set<AnyCancellable>
```

Reply:
@ducbm051291 I have also two separate story for [integration](https://github.com/Shayokh144/ic-survey-swiftui-combine/issues/7)  and [unit test](https://github.com/Shayokh144/ic-survey-swiftui-combine/issues/8). In this story I didn’t actually integrate just print the data for creating POW. I will integrate fully in the integration task. As the integration task can be a big one I created separate story for writing unit test. What do you think, should I create single story for writing unit test and integrating data?

Reply:
Nice one, so for the Backend task, I think you can cut off the code in ViewModel and save it for integration. What do you think?

Reply:
typealias added in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/43/commits/5430ab7c34e6d9db7b0812fb755363effc3354d9)